### PR TITLE
Pass arch to xbulk and sign all libs/freedoms

### PR DESCRIPTION
**Pull request by m-wynn**
Also don't mix tabs and spaces

### CODE REVIEW

**Comment by m-wynn** [comment]
This should allow for `xbps-mini-builder -a aarch64-musl` or passing of other args to xbulk (and thus some to xbps-src)

**Comment by the-maldridge** [comment]
Please split the whitespace changes to another commit.

File: xbps-mini-builder [hunk 1]
```diff
@@ -50,13 +50,13 @@ fi
...
-xbps-rindex --sign --signedby "XBPS-mini-builder" --privkey ../id_rsa hostdir/binpkgs
-xbps-rindex --sign-pkg --privkey ../id_rsa hostdir/binpkgs/*.xbps
+xbulk ${packages} "$@"
 
-if [ -d hostdir/binpkgs/nonfree ] ; then
-    xbps-rindex --sign --signedby "XBPS-mini-builder" --privkey ../id_rsa hostdir/binpkgs/nonfree
-    xbps-rindex --sign-pkg --privkey ../id_rsa hostdir/binpkgs/nonfree/*.xbps
-fi
+# Sign built packages
+for folder in {binpkgs binpkgs/multilib binpkgs/multilib/nonfree binpkgs/nonfree}; do
```
Comment (the-maldridge) [inline xbps-mini-builder]: move binpkgs to be part of the common path and then iterate on the remaining sub-directories.

**Comment by m-wynn** [comment]
Should be good now

**Pull request merged**